implants/lib/eldritch/src/file/list_impl.rs

```diff
@@ -0,0 +1,182 @@
+use anyhow::Result;
+use starlark::{values::{dict::Dict, Heap, Value}, collections::SmallMap, const_frozen_string};
+use sysinfo::{System, SystemExt, UserExt};
+use std::fs::DirEntry;
+#[cfg(unix)]
+use std::os::unix::fs::PermissionsExt;
+#[cfg(target_os = "macos")]
+use std::os::macos::fs::MetadataExt;
+#[cfg(target_os = "linux")]
+use std::os::linux::fs::MetadataExt;
+#[cfg(target_os = "windows")]
+use std::os::windows::fs::MetadataExt;
+use chrono::{Utc, DateTime, NaiveDateTime};
+use super::{File, FileType};
+
+const UNKNOWN: &str = "UNKNOWN";
+
+// https://stackoverflow.com/questions/6161776/convert-windows-filetime-to-second-in-unix-linux
+fn windows_tick_to_unix_tick(windows_tick: u64) -> i64{
```

Comment:
very based way back in time neither decided to use the same standard

Reply:
💯 Big fan.